<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lab1_3</class>
 <widget class="QMainWindow" name="lab1_3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>lab1_3</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>250</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Результат:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>161</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Вычислить</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Result">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>260</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCount">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>210</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Вычислить</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>137</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit_firstNum"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_SecNum"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_ThirdNum"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>145</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_1num">
       <property name="text">
        <string>Введите первое число:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2num">
       <property name="text">
        <string>Введите второе число:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3num">
       <property name="text">
        <string>Введите третье число:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>50</y>
      <width>137</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLineEdit" name="lineEdit_Anum"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Bnum"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>437</x>
      <y>50</y>
      <width>67</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label_Anum">
       <property name="text">
        <string>Введите A:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_Bnum">
       <property name="text">
        <string>Введите B:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
